package.json

```diff
@@ -19,6 +19,9 @@
       "<rootDir>/test/setup.js"
     ]
   },
+  "keywords": ["probot", "github-apps", "github", "pm", "automation", "robots", "workflow"],
```

Comment:
What is `pm`?

Reply:
Project management? It's a suggestion; happy to change it.

Reply:
or maybe remove it :)

Reply:
Removed!